tests/server/testServerWithBaseTheme.js

```diff
@@ -0,0 +1,109 @@
+/*
+Copyright 2019 OCAD University
+Licensed under the New BSD license. You may not use this file except in compliance with this licence.
+You may obtain a copy of the BSD License at
+https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENSE.txt
+*/
+
+
+"use strict";
+
+var fluid = require("infusion"),
+    kettle = require("kettle");
+
+require("../../src/server/staticHandlerBase");
+require("../../src/server/middleware/basicAuth");
+require("../../src/server/middleware/saveStoryWithBinaries");
+require("../../src/server/middleware/staticMiddlewareSubdirectoryFilter");
+require("../../src/server/dataSource");
+require("../../src/server/serverSetup");
+require("../../src/server/requestHandlers");
+require("./utils/serverTestUtils.js");
+
+kettle.loadTestingSupport();
+
+var sjrk = fluid.registerNamespace("sjrk");
+
+sjrk.storyTelling.server.testServerWithBaseThemeDefs = [{
+    name: "Base theme server tests",
+    expect: 10,
+    config: {
+        configName: "sjrk.storyTelling.server.testServerWithBaseTheme",
+        configPath: "./tests/server/configs"
+    },
+    components: {
+        clientConfigRequest: {
+            type: "kettle.test.request.http",
+            options: {
+                path: "/clientConfig",
+                method: "GET"
+            }
+        },
+        baseThemeFileBothExistRequest: {
+            type: "kettle.test.request.http",
+            options: {
+                path: "/testFile.txt",
+                method: "GET"
+            }
+        },
+        baseThemeFileBaseOnlyRequest: {
+            type: "kettle.test.request.http",
+            options: {
+                path: "/testBaseFile.txt",
+                method: "GET"
+            }
+        },
+        baseThemeFileCustomOnlyRequest: {
+            type: "kettle.test.request.http",
+            options: {
+                path: "/testCustomFile.txt",
+                method: "GET"
+            }
+        },
+        baseThemeFileMissingRequest: {
+            type: "kettle.test.request.http",
+            options: {
+                path: "/notARealFile.txt",
+                method: "GET"
+            }
+        }
+    },
+    sequence: [{
+        // test the clientConfig info endpoint
+        func: "{clientConfigRequest}.send"
+    }, {
+        event: "{clientConfigRequest}.events.onComplete",
+        listener: "sjrk.storyTelling.server.verifyGetClientConfigSuccessful",
+        args: ["{arguments}.0", "{arguments}.1", undefined]
```

Comment:
As said earlier, the fall back to the base theme should have already been kicked in here.